### PR TITLE
Updated to 1.9.0 and 1.9.4

### PR DESCRIPTION
**Pull request by NoxIceCat**
Really needed a great mod in a pack we are working on and other players along with myself have wanted it for a while.

Changed the chat import to TextFormatting replacing EnumChatFormatting and added some getters.

### CODE REVIEW

File: src/main/java/reauth/Main.java [hunk 1]
```diff
@@ -12,7 +12,7 @@
 import net.minecraftforge.fml.common.event.FMLPreInitializationEvent;
 import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
 
-@Mod(modid = "ReAuth", guiFactory = "reauth.GuiFactory", canBeDeactivated = true, clientSideOnly = true, acceptedMinecraftVersions = "[1.8.0,1.8.9]")
+@Mod(modid = "ReAuth", guiFactory = "reauth.GuiFactory", canBeDeactivated = true, clientSideOnly = true, acceptedMinecraftVersions = "[1.9.0,1.9.4]")
```
Comment (NoxIceCat) [inline src/main/java/reauth/Main.java]: Forgot to change the modid to lower case as server and client logs all say that modid's need to be lower case.

File: src/main/resources/mcmod.info [hunk 2]
```diff
@@ -1,14 +1,14 @@
 [
 {
-  "modid": "ReAuth",
```
Comment (NoxIceCat) [inline src/main/resources/mcmod.info]: Changed to lower case as server and client logs all say that modid's need to be lower case.

**Comment by TechnicianLP** [comment]
- looks interesting ... didnt notice anything about modids needing to be lowercase in the logs ... (i will look into that) ... 
- at the moment i am planning on a 1.9 version for it:
   - having some more security related things (securing stuff against other mods)(asm exclusions/ ...)
   - (possibly an eula but not sure on that one ...)

Another thing is i want to keep all of my branches at the same state (but 1.9 will be out soon)

but thanks for you work now i have less to do now for porting ;)

**Comment by NoxIceCat** [comment]
Its mainly when starting servers that I see it I was told by a fellow modder that I should always use lower case for this reason too.

Thanks for the updates though :)

**Comment by TechnicianLP** [comment]
1.9 should be avalible now

**Comment by NoxIceCat** [comment]
Thanks for the update I'll close this off now :)

**Pull request closed without merging**